spec/controllers/application_records_controller_spec.rb

```diff
@@ -47,6 +53,28 @@
     # No reason for staff to use this page
   end
 
+  describe 'GET #csv_export' do
+    before :each do
+      when_current_user_is :staff
+      @start_date = 1.week.ago.to_date
+      @end_date = Time.zone.today
```

Comment:
http://rails-bestpractices.com/posts/2014/10/22/use-time-zone-now-instead-of-time-now/